### DISCUSSION TITLE
some advises to adjust apisix dashbodrd UI，such as general plugins（traffic-plugin...） ，I hope we can adjuest parameter by UI  not  config。

### DISCUSSION THREAD

**Discussion by carverLiu** [apache/apisix#6090] [General]
some advises to adjust apisix dashbodrd UI，such as general plugins（traffic-plugin...） ，I hope we can adjuest parameter by UI  not  config。

**Comment by leslie-tsang** [comment]
cc @juzhiyuan PTAL

**Reply by juzhiyuan** [reply]
noted and cc @guoqqqi .

Also @carverLiu please take a look at https://docs.qq.com/doc/DTkdBcG5IT3V6VER3 I'm preparing the V3 proposal :)

**Reply by guoqqqi** [reply]
> such as general plugins（traffic-plugin...） ，I hope we can adjuest parameter by UI not config。

Thank you for your suggestion, I will take this task forward 😄

**Reply by juzhiyuan** [reply]
@guoqqqi kindly attach the issue link to this discussion :)
